numpy/core/tests/test_numeric.py

```diff
@@ -1541,43 +1541,6 @@ def test_array_double(self):
         act = self.clip(a, m, M)
         assert_array_strict_equal(ac, act)
 
-    def test_simple_nonnative(self):
-        # Test non native double input with scalar min/max.
-        # Test native double input with non native double scalar min/max.
-        a = self._generate_non_native_data(self.nr, self.nc)
-        m = -0.5
-        M = 0.6
-        ac = self.fastclip(a, m, M)
-        act = self.clip(a, m, M)
-        assert_array_equal(ac, act)
-
-        # Test native double input with non native double scalar min/max.
-        a = self._generate_data(self.nr, self.nc)
-        m = -0.5
-        M = self._neg_byteorder(0.6)
-        assert_(not M.dtype.isnative)
-        ac = self.fastclip(a, m, M)
-        act = self.clip(a, m, M)
-        assert_array_equal(ac, act)
-
-    def test_simple_complex(self):
```

Comment:
@birm can you please change the test so the new behavior is tested, rather than deleting the test?

Reply:
Ok. Sorry that I removed them, I'm not sure why I did. I'm sure there'll be style comments on the tests, so I'll squash my commits after that, if that's ok.